src/index.js

```diff
@@ -1059,7 +1059,16 @@ const _start = () => {
     if (u === '.') {
       console.warn('NOTE: You ran `exokit . <url>`\n(Did you mean to run `node . <url>` or `exokit <url>` instead?)')
     }
-    u = u.replace(/^exokit:/, '');
+    if (u.includes("exokit://")){
+      u = u.replace(/^exokit:\/\//, '');
+    }
+    if (u.includes("https//") || u.includes("http//")){
```

Comment:
This would not be valid so I don't see the need to check it.

Reply:
It is a case in discord when using `<exokit://https://test.com>` it turns it into `<exokit://https//test.com`.

Reply:
If it's a bug in discord, we should fix it there rather than adding magic to the engine. Is it possible to fix on the Discord end?

Reply:
Pending response from Discord.